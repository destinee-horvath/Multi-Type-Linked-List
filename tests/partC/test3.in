NEW 2
a
list
NEW 3
brr
its
cold
VIEW ALL
INSERT 0 1 {1}
NEW 1
a new list
NEW 3
rah
beep
blop
INSERT 2 0 {0}
DELETE 2 0
VIEW ALL
VIEW-NESTED 0
VIEW-NESTED 1
VIEW-NESTED 2
VIEW-NESTED 3