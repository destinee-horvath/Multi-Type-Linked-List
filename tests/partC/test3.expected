List 0: a -> list
List 1: brr -> its -> cold
Number of lists: 2
List 0
List 1
Nested 0: a -> {List 1} -> list
List 2: a new list
List 3: rah -> beep -> blop
INVALID COMMAND: INSERT
List 2: 
Number of lists: 4
Nested 0
List 1
List 2
List 3
a -> {brr -> its -> cold} -> list
brr -> its -> cold

rah -> beep -> blop
